rollup.config.js

```diff
@@ -0,0 +1,24 @@
+import ts from '@rollup/plugin-typescript'
+import { terser } from '@rollup/plugin-terser'
+
+export default {
+  input: 'src/main/minio.ts',
+  output: [
+    {
+      file: 'dist/main/minio.cjs',
+      format: 'cjs',
+      exports: 'default'
+    },
+    {
+      file: 'dist/main/minio.mjs',
+      format: 'es'
+    },
```

Comment:
Provide both ESM and CommonJS output, as the Javascript ecosystem is actually still on war between CJS vs ESM.

See:
- https://github.com/sindresorhus/meta/discussions/15
- https://gist.github.com/sindresorhus/a39789f98801d908bbc7ff3ecc99d99c